recipes/libcurl/all/conanfile.py

```diff
@@ -128,6 +137,13 @@ def requirements(self):
                 pass
             else:
                 self.requires("openssl/1.1.1g")
+        if self.options.with_wolfssl:
+            if tools.is_apple_os(self.settings.os) and self.options.darwin_ssl:
+                pass
```

Comment:
Shouldn't having `darwin_ssl` (or `with_winssl`) **and** `with_wolfssl` raise a `ConanInvalidConfiguration` as it does in the changes above?